test/cypress/e2e/search.spec.js

```diff
@@ -134,18 +122,27 @@ describe('Search', () => {
    */
   it('Search without signing in - happens in mainnet', () => {
     cy.visit(urls.dashboard);
-    cy.get(ss.searchInput).click().type(`${accounts['mainnet delegate'].address}{enter}`);
+    cy.get(ss.searchIcon).click();
+    cy.get(ss.searchInput).type(`${accounts['mainnet delegate'].address}`);
+    cy.wait('@requestAccount');
+    cy.get(ss.searchAccountRow).eq(0).click();
     cy.get(ss.accountName).should('have.text', accounts['mainnet delegate'].username);
   });
 
   /**
    * Search signed in mainnet
    * @expect happens in mainnet
+   * This test should be fxied once the transaction details page
+   * change the behavior of how to present the data
```

Comment:
Ticket number? Bit more specific? I dont know what change are you talking about. Is it broken now?